### DISCUSSION TITLE
Issue with the image upload

### DISCUSSION THREAD

**Discussion by P-Neha** [getgauge/taiko#2006] [Development]
There are two images in a page. One image is getting uploaded and when the second image has to be uploaded it's failing and giving a timed out error as below. I'm using the below attach command. I'm not understanding why the same command is failing for second image.  

await attach('./image/' + heroImageName, fileField({ id: 'hero-image-upload' }), { force: true });

FYI: For the first image command is the same and just the "id" is different

Error Message: Error: Timed out
        Stacktrace: 
        Error: Timed out
            at Timeout.<anonymous> (C:\Users\AppData\Roaming\gauge\plugins\js\2.3.15\src\test.js:44:23)
            at listOnTimeout (internal/timers.js:554:17)
            at processTimers (internal/timers.js:497:7)

**Comment by P-Neha** [comment]
Any help is highly appreciated! Thank you

**Comment by NivedhaSenthil** [comment]
This timeout seems to be happening from Gauge.. try increasing the step timeout `test_timeout` in `env/default/js.properties` file

**Reply by P-Neha** [reply]
Thanks much! That was really helpful.
